### ISSUE TITLE
flags --set-exit-code-on-error  --set-exit-code-below-score are not available in 0.2.1 when installed thorough brew

### ISSUE THREAD

**Issue by kdhulipala** [FairwindsOps/polaris#161]
Was trying out 0.2.1 installed through Homebrew as suggested on the Readme. Wanted to try failing polaris in CI/CD when there are any error-level issues hence followed the command with flags advised.
```
polaris --audit --audit-path ./deploy/ \
  --set-exit-code-on-error \
  --set-exit-code-below-score 90
```

However got the error `flag provided but not defined: -set-exit-code-on-error`. 
```
$ polaris --version
Polaris version 0.2.1
```
```
$ polaris --help
Usage of polaris:polaris
  -audit
    	Runs a one-time audit.
  -audit-path string
    	If specified, audits one or more YAML files instead of a cluster
  -config string
    	Location of Polaris configuration file
  -dashboard
    	Runs the webserver for Polaris dashboard.
  -dashboard-base-path string
    	Path on which the dashboard is served (default "/")
  -dashboard-port int
    	Port for the dashboard webserver (default 8080)
  -disable-webhook-config-installer
    	disable the installer in the webhook server, so it won't install webhook configuration resources during bootstrapping
  -display-name string
    	An optional identifier for the audit
  -kubeconfig string
    	Paths to a kubeconfig. Only required if out-of-cluster.
  -log-level string
    	Logrus log level (default "info")
  -master string
    	The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.
  -output-file string
    	Destination file for audit results
  -output-format string
    	Output format for results - json, yaml, or score (default "json")
  -output-url string
    	Destination URL to send audit results
  -version
    	Prints the version of Polaris
  -webhook
    	Runs the webhook webserver.
  -webhook-port int
    	Port for the webhook webserver (default 9876)
```

Did I miss anything?

**Comment by bobby-brennan** [comment]
Thanks! We haven't yet cut a release with the new functionality. The release is in progress now:

https://github.com/FairwindsOps/polaris/pull/162

**Comment by bobby-brennan** [comment]
This should be available now in `0.3.0`. Thanks for the report!
